modules/services/parcellite.nix

```diff
@@ -0,0 +1,52 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.parcellite;
+
+in
+
+{
+  meta.maintainers = [ maintainers.gleber ];
+
+  options = {
+    services.parcellite = {
+      enable = mkEnableOption "Parcellite";
+
+      package = mkOption {
+        type = types.package;
+        default = pkgs.parcellite;
+        defaultText = "pkgs.parcellite";
+        description = "Parcellite package to install.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    home.packages = [ cfg.package ];
+
+    systemd.user.services.parcellite = {
+      Unit = {
+        Description = "Lightweight GTK+ clipboard manager";
+        After = [ "graphical-session-pre.target" ];
+        PartOf = [ "graphical-session.target" ];
+      };
+
+      Install = {
+        WantedBy = [ "graphical-session.target" ];
+      };
+
+      Service = {
+        # Ideally PATH entries should be embedded in nixpkgs using wrapProgram.
+        # XDG_DATA_DIRS is necessary to make it pick up icons correctly.
+        Environment = ''
+          PATH=${cfg.package}/bin:${pkgs.which}/bin:${pkgs.xdotool}/bin XDG_DATA_DIRS=${pkgs.hicolor_icon_theme}/share
```

Comment:
Yeah, I think this belongs in Nixpkgs. I created https://github.com/NixOS/nixpkgs/pull/32364 for this.

Reply:
Ack. If I read your comment correctly, I am keeping Environment for compatibility with older nixpkgs which do not have the PR merged.